src/genie/libs/parser/dell/tests/ShowVersion/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,21 @@
+expected_output = {
+    'version': {
+        'machine_dsc': 'Dell Networking Switch',
+        'model': 'N1548',
+        'machine_type': 'Dell Networking N1548',
+        'serial': 'CN0V143P2829856D0183A00',
+        'manufacturer': '0xbc00',
+        'bia': 'F8B1.5683.8731',
+        'sys_obj_id': '1.3.6.1.4.1.674.10895.3065',
+        'soc_ver': 'BCM56150_A0',
+        'hw_ver': '2',
+        'cpld_ver': '16',
+        'versioning': {
```

Comment:
not sure what should be... but it looks like below is better...?
```
'unit': {
     '1': {
        'active_ver': '6.3.2.4',
```
assume the case which has multiple units in the output...